packages/contracts-bedrock/tasks/check-l2.ts

```diff
@@ -577,6 +577,9 @@ const check = {
 
     const addressManager = await ProxyAdmin.addressManager()
     console.log(`  - addressManager: ${addressManager}`)
+
+    await checkProxy(hre, 'ProxyAdmin')
+    await assertProxy(hre, 'ProxyAdmin')
```

Comment:
```suggestion
    await checkProxy(hre, 'ProxyAdmin', signer.provider)
    await assertProxy(hre, 'ProxyAdmin', signer.provider)
```

Reply:
^^ This is correct, will break when using a URL passed via CLI if not

Reply:
CI failing with this error:

```
tasks/check-l2.ts(581,11): error TS2554: Expected 3 arguments, but got 2.
```